### PR TITLE
Migration from the collection per entity to the collection per service path data model

### PR DESCRIPTION
**Pull request by gtorodelvalle**
- Implements https://github.com/telefonicaid/fiware-sth-comet/issues/311
- Fixes https://github.com/telefonicaid/fiware-sth-comet/issues/249
- No linting issues
- 100% tests passed

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -355,6 +356,56 @@ parameter. This means that depending on its value, only the associated data (raw
...
+    -c, --collection <collectionName>  only this collection will be taken info consideration, a database is mandatory if a collection is set
+```
+
+Special care should be taken when requesting a data model migration since the migration of aggregated data is not an idempotent operation if the target data model collections already exist. In this case, the already existent data stored in these collections is combined with the one stored in the original data model collections pending migration. Based on this fact, we suggest the next procedure when considering a data model migration:
+
+1. Check the current configured data model used by the STH component based on the options detailed in [Running the STH server](#section5) below. Typically it will be the default collection per entity data model.
+2. Get a data model analysis report about the databases and collections which need to be migrated to the new desired data model running the following command: `LOGOPS_FORMAT=dev DATA_MODEL=collection-per-service-path ./bin/sth_database_model -a`. Use the desired data model instead of the one included in the previous example.
```
Comment (dmoranj) [inline README.md]: Move this command to a code textbox, like:
```
`LOGOPS_FORMAT=dev DATA_MODEL=collection-per-service-path ./bin/sth_database_model -a`
```
to improve readability.

Comment (dmoranj) [inline README.md]: Same for the rest of commands below.

Comment (gtorodelvalle) [inline README.md]: Fixed in https://github.com/telefonicaid/fiware-sth-comet/pull/312/commits/61fb711d0bfd7dc2855678322a6b1a8882a2fb67

**Comment by dmoranj** [comment]
LGTM

**Pull request merged**